sapi/apache2handler/sapi_apache2.c

```diff
@@ -461,11 +461,14 @@ static int php_pre_config(apr_pool_t *pconf, apr_pool_t *plog, apr_pool_t *ptemp
 	return OK;
 }
 
+ZEND_API extern bool use_deepbind;
```

Comment:
Wouldn't work if this was wrapped to some functio (e.g. `zend_use_deepbind()` that would set such variable in zend code). I don't mind the Zend part but it's not exactly nice API for SAPI...